internal/mesh/proxy-tracker/proxy_state_exports.go

```diff
@@ -34,9 +35,13 @@ func (p *ProxyState) AllowEmptyClusters() bool {
 }
 
 func (p *ProxyState) Authorize(authz acl.Authorizer) error {
-	// TODO(proxystate): we'll need to implement this once identity policy is implemented
-
-	// Authed OK!
+	// authorize for mesh proxies.
+	// TODO(proxystate): implement differently for gateways
+	allow := authz.ToAllowAuthorizer()
+	if err := allow.IdentityWriteAllowed(p.Identity.Name, resource.AuthorizerContext(p.Identity.Tenancy)); err != nil {
+		return err
+	}
+	
```

Comment:
~ Looks like maybe discrepancy between Ent and this change due to whitespace, which may also be impacting the linter errors?

Reply:
yes.  thanks.  something got off with my commits and the script did not work, so I had to generate this by hand.  I will fix.  Thanks for pointing this out!